<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="font">
    <font>
     <family>Courier</family>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <widget class="QWindowsBar" name="windows_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>602</y>
      <width>1060</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1071</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="timelabel">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCuDesktopWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QDesktopIcon" name="icon_trash" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_system" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_todo" native="true">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>80</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_hintsfolder" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_notepad" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_webchat" native="true">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>0</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_notefolder" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_update" native="true">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>80</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QDesktopIcon" name="icon_help" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>64</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <zorder>icon_trash</zorder>
    <zorder>icon_system</zorder>
    <zorder>icon_todo</zorder>
    <zorder>icon_hintsfolder</zorder>
    <zorder>icon_notepad</zorder>
    <zorder>icon_webchat</zorder>
    <zorder>icon_notefolder</zorder>
    <zorder>icon_update</zorder>
    <zorder>icon_help</zorder>
    <zorder>windows_menu</zorder>
    <zorder>horizontalLayoutWidget</zorder>
   </widget>
   <widget class="QWidget" name="MenuBar" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>387</y>
      <width>141</width>
      <height>215</height>
     </rect>
    </property>
    <widget class="QFrame" name="TextFrame">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>24</width>
       <height>210</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="VerticalLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>60</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>95</string>
      </property>
     </widget>
     <widget class="VerticalLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>90</y>
        <width>31</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Windoors</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="ButtonFrame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>-3</y>
       <width>111</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>7</y>
        <width>121</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="menuVerticalLayout" stretch="0,0,0,0,0,0,0,0">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_prog">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Programs         ▶</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_docs">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Documents      ▶</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sett">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Settings           ▶</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_find">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Find                 ▶</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_help">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_run">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Run...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_shut">
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Shut Down...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDesktopIcon</class>
   <extends>QWidget</extends>
   <header>qdesktopicon.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCuDesktopWidget</class>
   <extends>QWidget</extends>
   <header>qcudesktopwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VerticalLabel</class>
   <extends>QLabel</extends>
   <header>verticallabel.h</header>
  </customwidget>
  <customwidget>
   <class>QWindowsBar</class>
   <extends>QFrame</extends>
   <header>qwindowsbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
